### DISCUSSION TITLE
Getting into Aves through camera app.

### DISCUSSION THREAD

**Discussion by Pandemic666** [deckerst/aves#817] [General]
hi all, I use galaxy a54 one ui 6. I want to hop into Aves through camera. when I click on photo, the gallery, on bottom left corner it says: Unable to find application to perform this action.

**Comment by deckerst** [comment]
Please install this app: https://github.com/MuntashirAkon/AppManager/
Then try again to open the image from your camera app. You should see the `AppManager` app as an option, and that app should show you what the camera image opening intent is made of. Please screenshot all the intent details and paste it here.

**Reply by Pandemic666** [reply]
thanks for the reply. I have installed appmanager and tried to open the image but it still prompts me "Unable to find application to perform this action."

**Reply by deckerst** [reply]
Has it ever worked with any other app? I'm surprised it does not even suggest the preinstalled gallery app as an option.
Is your camera app the preinstalled Samsung Camera, or something else?

**Reply by Pandemic666** [reply]
it is samsung camera. I disabled the gallery app, I only want to use aves

**Reply by deckerst** [reply]
Ah, that's why. The Samsung Camera only allows opening with the Samsung Gallery. It sucks, but that's Samsung decision.

**Reply by Pandemic666** [reply]
so there is no solution to this?

**Reply by deckerst** [reply]
Yes, it's a limitation of Samsung Camera.

**Reply by deckerst** [reply]
If you ever find an app that finds a way to work around that limitation, I'm interested.

**Reply by Pandemic666** [reply]
I see, thanks for the Aves Im in love with it! keep up the good work
